packages/frontend/src/pages/Contribute.jsx

```diff
@@ -47,7 +47,16 @@ export default observer(() => {
     <>
       <div className="contribute-container">
         <div className="contribute-left">
-          <Link to="/">
+          <Link
+            to="/"
+            style={{
+              pointerEvents:
+                contributeState === contributeState.queueing ||
+                contributeState === contributeState.contributing
+                  ? 'none'
+                  : '',
+            }}
+          >
```

Comment:
I'm not able to get any version of this condition to work, can you see why?
Trying to make the link back to home disabled after the user has begun the contribution process, but still allow if they haven't started yet. People may want to click around the different pages before they start.
The link is still working here even in queueing and contributing states.

Reply:
Since it is `ContributeState.queueing` instead of `contributeState.queueing`, same as the `contributing`, I already fix it and will directly merge to my branch :)

Reply:
Since it is `ContributeState.queueing` instead of `contributeState.queueing`, same as the `contributing`, I already fix it and will directly merge to my branch :)